src/utils/setupdb.js

```diff
@@ -0,0 +1,46 @@
+import mongoose from 'mongoose';
+
+/**
+ * Connect to a mongoose database
+ *
+ * @param {string} mongoDbUri the URI of the MongoDB to connect too. Including credentials
+ * @param {string} databaseName the database to switch to on connection
+ * @returns {Mongoose} the Mongoose API setup for this application.
+ */
+export default (mongoDbUri, databaseName) => {
+  const connectionString = createConnString(mongoDbUri, databaseName);
+
+  // Create the database connection
+  mongoose.connect(connectionString, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
+
+  // Log successful connection
+  mongoose.connection.on('connected', function () {
```

Comment:
Unless we _need_ to use declare a function by keyword, I'd prefer that we use rocket functions 🙂